src/cirrus/build.py

```diff
@@ -121,6 +129,41 @@ def execute_build(opts):
     local('. ./{0}/bin/activate && python setup.py develop'.format(venv_name))
 
 
+def build_docs(opts):
+    """
+    _build_docs_
+
+    Runs 'make' against a Sphinx makefile.
+    Requires the following cirrus.conf section:
+
+    [doc]
+    sphinx_makefile_dir = /path/to/makefile
+
+    : param argparse.Namspace opts: A Namespace of build options
+    """
+    LOGGER.info('Building docs')
+    config = load_configuration()
+
+    try:
+        docs_root = os.path.join(os.getcwd(),
+                                 config['doc']['sphinx_makefile_dir'])
+    except KeyError:
+        LOGGER.error('Did not find a complete [doc] section in cirrus.conf'
+                     '\nSee below for an example:'
+                     '\n[doc]'
+                     '\n;sphinx_makefile_dir = /path/to/sphinx')
+        sys.exit(1)
+
+    cmd = 'cd {} && make clean html'.format(docs_root)
+
+    if opts.docs:
+        # additional args were passed after --docs.  Pass these to make
+        cmd = 'cd {} && make {}'.format(docs_root, ' '.join(opts.docs))
+
+    local(cmd)
```

Comment:
@ksnavely this is run in another process and will not change the directory from where you ran the script.